examples/array-map-symmetry/array_map_symmetry.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina/io;
+
+public function main() {
+    // constructors for arrays and maps
```

Comment:
IMO, these comments don't quite add value. We can probably say "List constructors are used to construct list values". And then have a similar comment for the mapping constructor.